README.md

```diff
@@ -248,6 +232,46 @@ Kubectl job:
     - kubectl version --client
 ```
 
+## [.base_notify_github_stage](./base_notify_github_stage.yml)
+
+use [@SocialGouv/azure-db](https://github.com/SocialGouv/docker/tree/master/azure-db) docker image
+
+```yaml
+include:
+  - project: SocialGouv/gitlab-ci-yml
+    file: /base_notify_github_stage.yml
+    ref: v4.0.1
+#
+
+Notify Starting Deployment:
+  extends: .base_notify_pending_stage
+  stage: Deploy
+  except:
+    - /^renovate/*
+
+Notify Fail:
+  extends: .base_notify_fail_stage
+  stage: Notify Finished Deployment
+  except:
+    - /^renovate/*
+  dependencies:
+    - Notify Starting Deployment
+  before_script:
+    - source ./.gitlab-ci/env.sh
+    - HOST=${FRONTEND_URL}
+
+Notify Success:
+  extends: .base_notify_success_stage
+  stage: Notify Finished Deployment
+  except:
+    - /^renovate/*
+  dependencies:
+    - Notify Starting Deployment
+  before_script:
+    - source ./.gitlab-ci/env.sh
```

Comment:
this is not mandatory. There is lots of other ways to set up the env.
Only le line below `HOST=https://example.com` is important